src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Properties/Resources.Designer.cs

```diff
@@ -79,83 +79,74 @@ public class Resources {
         /// </summary>
-        public static string Microsoft_plugin_folder_truncation_warning_title {
+        public static string Microsoft_plugin_folder_open_containing_folder {
             get {
-                return ResourceManager.GetString("Microsoft_plugin_folder_truncation_warning_title", resourceCulture);
+                return ResourceManager.GetString("Microsoft_plugin_folder_open_containing_folder", resourceCulture);
             }
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Add.
+        ///   Looks up a localized string similar to Folder: Use &gt; to search within the directory. Use * to search for file extensions. Or use both &gt;*..
         /// </summary>
-        public static string wox_plugin_folder_add {
+        public static string Microsoft_plugin_folder_open_current_folder_subtitle {
             get {
-                return ResourceManager.GetString("wox_plugin_folder_add", resourceCulture);
+                return ResourceManager.GetString("Microsoft_plugin_folder_open_current_folder_subtitle", resourceCulture);
             }
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Delete.
+        ///   Looks up a localized string similar to Open path in console (Ctrl+Shift+C).
         /// </summary>
-        public static string wox_plugin_folder_delete {
+        public static string Microsoft_plugin_folder_open_in_console {
             get {
-                return ResourceManager.GetString("wox_plugin_folder_delete", resourceCulture);
+                return ResourceManager.GetString("Microsoft_plugin_folder_open_in_console", resourceCulture);
             }
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Are you sure you want to delete {0}?.
+        ///   Looks up a localized string similar to Could not start .
         /// </summary>
-        public static string wox_plugin_folder_delete_folder_link {
+        public static string Microsoft_plugin_folder_start_failed {
             get {
-                return ResourceManager.GetString("wox_plugin_folder_delete_folder_link", resourceCulture);
+                return ResourceManager.GetString("Microsoft_plugin_folder_start_failed", resourceCulture);
             }
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Edit.
+        ///   Looks up a localized string similar to Showing {0} of {1} results.
         /// </summary>
-        public static string wox_plugin_folder_edit {
+        public static string Microsoft_plugin_folder_truncation_warning_subtitle {
             get {
-                return ResourceManager.GetString("wox_plugin_folder_edit", resourceCulture);
+                return ResourceManager.GetString("Microsoft_plugin_folder_truncation_warning_subtitle", resourceCulture);
```

Comment:
We should add comments in the resource file for cases like this. Ideally the comment should describe the meaning of the phrase and where it could appear for more context, and in the case we would also mentioned that {0} is a placeholder for number of results found and {1} is for total number of results.

Reply:
Sounds good. Maybe we want to add that to the localization issue if it's not already present.